styles/antarctica.css

```diff
@@ -4,3 +4,21 @@
         height: auto;
       } */
 
+.antarcticaImages {
+    width: 100%;
+    height: 100%;
```

Comment:
Giving a percentage will tell the image to fill the container to 100%. If you assign either the height or with width 100% and assign the opppsite auto it will help shrink your images. Ensure your container has a defined size. Otherwise, your container will grow with the image.

Reply:
thanks Nick.